client/asset/bch/spv.go

```diff
@@ -477,6 +476,8 @@ func (w *bchSPVWallet) Stop() {
 	}
 
 	w.log.Info("SPV wallet closed")
+	// Allow the logger to be re-initiated when the wallet reconnects.
+	atomic.CompareAndSwapUint32(&loggingInited, 1, 0)
```

Comment:
nit: Can just do [StoreUint32(addr *uint32, val uint32)](https://pkg.go.dev/sync/atomic#StoreUint32) since you are not using the return.

Reply:
Same thought.  Not using the return has the smell of a bug